src/Microsoft.AspNet.Mvc.Core/Rendering/Html/TemplateRenderer.cs

```diff
@@ -82,6 +84,8 @@ internal class TemplateRenderer
             _viewData = viewData;
             _templateName = templateName;
             _readOnly = readOnly;
+            _logger = _viewContext.HttpContext.RequestServices.GetRequiredService<ILoggerFactory>()
+                .CreateLogger<TemplateRenderer>();
```

Comment:
I actually wanted to have template-view related logging for scenarios where users have custom  templates (ex: Customer display/edit template for model Customer) and in case of any failure where the template for the model is not found, we could log the info having the search locations...but the issue is that this method even is called for all the properties on a model when we probe for display/edit templates, which creates lot of entries in the log...
I am not too strong about this...will remove it in next iteration...